webpack/webpack5-localization-plugin/src/AssetProcessor.ts

```diff
@@ -267,7 +274,8 @@ function _reconstructNonLocalized(
 ): INonLocalizedReconstructionResult {
   const issues: string[] = [];
 
-  for (const element of reconstructionSeries) {
+  for (let i: number = reconstructionSeries.length - 1; i >= 0; i--) {
```

Comment:
NIT: For my own knowledge, does using a standard for-loop vs for-of-loop provide any perf benefits? Asking because I don't see the `i` get used elsewhere and I find for-of to be ever-so-slightly more readable

Reply:
This was actually about doing this in reverse order since if you iterate replacements in reverse order you don't have to tamper with offsets, but I just looked at ReplaceSource and it iterates in ascending order, so I'll revert this part.

Reply:
Reverted.